packages/contracts-bedrock/scripts/upgrades/holocene/templates/bundle_template.json

```diff
@@ -0,0 +1,67 @@
+{
+  "chainId": $L1_CHAIN_ID,
+  "metadata": {
+    "name": "Holocene Hardfork - Contracts Upgrade",
+    "description": "Upgrades the `SystemConfig.sol`, `MIPS.sol`, `FaultDisputeGame.sol`, and `PermissionedDisputeGame.sol` contracts for Holocene."
+  },
+  "transactions": [
+    {
+      "metadata": {
+        "name": "Upgrade `SystemConfig` proxy",
+        "description": "Upgrades the `SystemConfig` proxy to the new implementation, featuring configurable EIP-1559 parameters."
```

Comment:
Confifming there are no changes to the initializer that require first using the StorageSetter, then reinitializing?

Reply:
No need. The EIP-1559 params are expected to be zero upon the upgrade. See [here](https://github.com/ethereum-optimism/optimism/pull/12878#discussion_r1833558831).